.github/workflows/conformance_with_build.yml

```diff
@@ -31,27 +31,27 @@ jobs:
     strategy:
       matrix:
         config:
-          - name: "consul@v1.11 + consul-k8s@v0.49.0"
-            api-gateway-image: "consul-api-gateway:local-build"
-            consul-image: "hashicorp/consul:1.11"
-            envoy-image: "envoyproxy/envoy:v1.20-latest"
-            consul-k8s-version: "v0.49.0"
-          - name: "consul@v1.12 + consul-k8s@v0.49.0"
+          - name: "consul@v1.12 + consul-k8s@v0.49.1"
             api-gateway-image: "consul-api-gateway:local-build"
             consul-image: "hashicorp/consul:1.12"
             envoy-image: "envoyproxy/envoy:v1.22-latest"
-            consul-k8s-version: "v0.49.0"
-          - name: "consul@v1.13 + consul-k8s@v0.49.0"
-            api-gateway-image: "hashicorppreview/consul-api-gateway:0.4-dev"
```

Comment:
This looks like it was an inadvertent change at some point, likely a copy-paste error, where we stopped actually testing the local build on the `conformance_with_build` matrix.